app/index.js

```diff
@@ -155,19 +156,24 @@ var CapitalFrameworkGenerator = yeoman.generators.Base.extend({
     },
 
     processReadme: function() {
+        this.readme = this.readFileAsString( path.join( this.sourceRoot(), '_README.md' ) );
+        this.readme = this.engine( this.readme, this );
+
         // The README that comes from OSPT doesn't include template variables so
         // we have to manually regex what we want out of it.
-        var readme = this.readFileAsString( this.destinationRoot() + '/_cache/open-source-project-template-master/README.md'),
```

Comment:
:tada:

Love that this replaces the regex with something that's easier to edit and maintain.